### PR TITLE
Fix typo in localized warning message

### PR DESCRIPTION
**Pull request by mehalick**
**Task name**: DotNetCoreCLIV2

**Description**: Fixed spelling mistake "requied" to "required"

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) #13270 

**Checklist**:
- [ ] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -135,5 +135,5 @@
...
-  "loc.messages.DeprecatedDotnet2_2_And_3_0": ".NET Core SDK/runtime 2.2 and 3.0 are now End of Line(EOL) and have been removed from all agents. If you're using these SDK/runtimes, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the requied version."
-}
\ No newline at end of file
+  "loc.messages.DeprecatedDotnet2_2_And_3_0": ".NET Core SDK/runtime 2.2 and 3.0 are now End of Line (EOL) and have been removed from all agents. If you're using these SDK/runtimes, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version."
```
Comment (bgrainger) [inline Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson:138]:
```suggestion
  "loc.messages.DeprecatedDotnet2_2_And_3_0": ".NET Core SDK/runtime 2.2 and 3.0 are now End of Life (EOL) and have been removed from all agents. If you're using these SDK/runtimes, kindly upgrade to newer versions which are not EOL, or else use UseDotNet task to install the required version."
```

`End of Line` should be `End of Life`; cf. https://dotnet.microsoft.com/platform/support/policy/dotnet-core

**Comment by mehalick** [comment]
@bgrainger The phrase "End of Line" has been corrected to "End of Life"; I wasn't aware this was also a mistake and I'm happy to fix as part of this PR. Thanks for the suggestion

**Comment by mehalick** [comment]
@bgrainger @bishal-pdMSFT @tejasd1990 It appears this PR now duplicates work done in #13282 and I recommend declining this PR and closing my original issue. If I can be of any assistance certainly let me know, thanks

**Comment by bishal-pdMSFT** [comment]
Yes @mehalick, thanks for creating the PR which made us aware of this typo. But since we were anyways making another change in the string, we took this change along. Hope that is fine!

**Pull request closed without merging**